docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debezium/connect:latest
+
+USER root
+
+RUN microdnf -y install git java-11-openjdk-devel
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
+ENV KAFKA_HOME /kafka
+RUN export JAVA_HOME
```

Comment:
This is unnecessary; the `ENV` directive above already makes this variable available to subsequent commands (during both the build and at runtime).